lib/Doctrine/ORM/Configuration.php

```diff
@@ -676,7 +677,7 @@ public function getFilterClassName($name)
      *
      * @return void
      *
-     * @throws ORMException If not is a \Doctrine\Common\Persistence\ObjectRepository
+     * @throws ORMException If not is a ObjectRepository.
```

Comment:
```suggestion
     * @throws ORMException If not is an ObjectRepository.
```

Reply:
I didn't notice until now, but this is still a bit weird :sweat_smile: